sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -58,8 +58,12 @@ extern "C" {
  * @note    Only applicable with
  *          [gnrc_sixlowpan_frag_fb](@ref net_gnrc_sixlowpan_frag_fb) module
  */
-#ifndef CONFIG_GNRC_SIXLOWPAN_FRAG_FB_SIZE
+#ifndef GNRC_SIXLOWPAN_FRAG_FB_SIZE
+#if defined(MODULE_GNRC_SIXLOWPAN_FRAG_SFR)
+#define CONFIG_GNRC_SIXLOWPAN_FRAG_FB_SIZE         (4U)
+#else   /* defined(MODULE_GNRC_SIXLOWPAN_FRAG_SFR) */
 #define CONFIG_GNRC_SIXLOWPAN_FRAG_FB_SIZE         (1U)
+#endif  /* defined(MODULE_GNRC_SIXLOWPAN_FRAG_SFR) */
 #endif
```

Comment:
@leandrolanzieri @PeterKietzmann I guess I need to adapt this for Kconfig somehow? What do I need to do?

Reply:
This behaviour in `sys/net/gnrc/network_layer/sixlowpan/frag/fb/Kconfig` could be:
```
config GNRC_SIXLOWPAN_FRAG_FB_SIZE
    int "Number of datagrams that can be fragmented simultaneously"
    default 4 if MODULE_GNRC_SIXLOWPAN_FRAG_SFR
    default 1
    help
        This determines the number of @ref gnrc_sixlowpan_frag_fb_t instances
        available.
```
Is that what you refer to?

Reply:
Ok, but then I also need to provide the `gnrc_sixlowpan_frag_sfr` config first, right?

Reply:
All the `MODULE_` symbols are generated from `USEMODULE`, so just by using `gnrc_sixlowpan_frag_sfr` module it will take 4 as default.